app/uk/gov/hmrc/economiccrimelevyreturns/views/AmountDueView.scala.html

```diff
@@ -88,7 +88,18 @@ <h2 class="govuk-heading-m">@messages("amountDue.bands")</h2>
 
         <p class="govuk-body">@messages("amountDue.bands.p1")</p>
         <p class="govuk-body">@messages("amountDue.bands.p2")</p>
-        <p class="govuk-body">@messages("amountDue.bands.p3")</p>
+
+        @if(calculatedLiability.amountDue.amount == 0) {
```

Comment:
```suggestion
        @if(calculatedLiability.calculatedBand == Small) {
```
Suggest this change just to be consistent with logic elsewhere in this page for a nil return (see line 66).